Makefile.include

```diff
@@ -399,6 +399,7 @@ ifneq (,$(IOTLAB_NODE))
 endif
 
 # Add standard include directories
+INCLUDES += -I$(APPDIR)
```

Comment:
Thanks for fixing this! I have falling into this pitfall numerous times when I needed to quickly test something, but never really traced it down but instead did a quick-and-dirty hack of the system header instead.

I'm glad that next time I need to quickly test something, I won't fall into that pitfall again :heart: